sea-rt/seahorn.cpp

```diff
@@ -166,30 +169,81 @@ void __assert_fail (const char * assertion, const char * file,
 }
 
 
-void __sea_set_extptr_slot0(void* ptr, uint32_t base) {
+void* __sea_set_extptr_slot0_fp(void* ptr, size_t base) {
```

Comment:
if this implementation is not working, should comment that it is just a draft. 

Probably a good idea to split `seahorn.cpp` into multiple files at this point